apiclient/templates/typescript/axios/unstable.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+java -jar openapi-generator-cli.jar generate \
+    --input-spec $(System.ArtifactsDirectory)/openapi/openapi.json \
+    --generator-name typescript-axios \
+    --output ./apiclient/generated/typescript/axios  \
+    --template-dir ./apiclient/templates/typescript/axios \
+    --ignore-file-override ./apiclient/.openapi-generator-ignore \
+    --additional-properties=useSingleRequestParameter="true",npmName="axios",snapshotVersion="-SNAPSHOT.$(Build.BuildNumber)",npmRepository="https://dev.azure.com/jellyfin-project/jellyfin/_packaging"
```

Comment:
The actual repo url is `https://pkgs.dev.azure.com/jellyfin-project/jellyfin/_packaging/unstable/npm/registry/ `